libs/orders/src/lib/order-hooks/use-order-validation.tsx

```diff
@@ -88,65 +81,13 @@ export const useOrderValidation = ({
       };
     }
 
-    if (market.state !== MarketState.Active) {
-      if (market.state === MarketState.Suspended) {
-        if (market.tradingMode === MarketTradingMode.Continuous) {
```

Comment:
this was wrong. continuous trading (i.e. not in auction) would keep the market in the active state.